src/components/components/map/map.js

```diff
@@ -2,20 +2,49 @@ import './map.scss';
 
 import React from 'react';
 import { Map, Marker, Popup, TileLayer } from 'react-leaflet';
-
 import { ClubTeaser } from './../club-teaser-list/club-teaser-list';
 
 // local modules
 // styles
 class ClubMap extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      clubs: this.props.clubs || [],
+      bounds: [[25.3345123, -99.5218668], [38.3345123, -80.5218668]]
+    };
+  }
+
+  componentDidMount = () => {
+    this.props.clubs !== this.state.clubs && this.updateClubs(this.props.clubs);
+  };
+
+  componentDidUpdate() {
+    this.props.clubs !== this.state.clubs && this.updateClubs(this.props.clubs);
+  }
+
+  updateClubs = clubs => {
+    let boundsArray = [];
+    this.props.clubs.map((club, i) => {
```

Comment:
map() returns a new array, can't this be merged with the above line? 

Also I think that i isn't used here